db/migrate/20220817123038_portfolio_automation.rb

```diff
@@ -0,0 +1,6 @@
+class PortfolioAutomation < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :portfolio_auto_generated, :boolean, default: false, null: false
+    add_column :units, :portfolio_auto_generation_date, :datetime
```

Comment:
Do we default this to 2 weeks after the draft learning summary report for the existing units that have it?

Reply:
And the unit is not over ... so now < unit end date

Reply:
> Do we default this to 2 weeks after the draft learning summary report for the existing units that have it?

What about units that doesn't have it? Since this is a failsafe, how about we just default it to 3 days before unit end date to make it simpler?

> And the unit is not over
I would rather not have this as null if possible, end date - 3 should be a safe value to apply to all units, including inactive ones.